sdk/go/common/workspace/plugins.go

```diff
@@ -995,6 +1004,30 @@ type PluginSpec struct {
 	Checksums map[string][]byte
 }
 
+func NewAnalyzerPlugin(name tokens.QName, version *semver.Version) PluginSpec {
+	return PluginSpec{
+		Name:    strings.ReplaceAll(string(name), tokens.QNameDelimiter, "_"),
```

Comment:
Do we actually need these ReplaceAlls? Where are we getting plugin names with slashes in the first place from?

Reply:
I'm not sure tbh.  But they were already there, so I didn't want to change too many things at once in this PR.